src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -94,7 +94,7 @@ function VotingActions({ canJurorVote, onRequestCommit }) {
           onClick={() => onRequestCommit(VOTE_OPTION_REFUSE)}
           width={buttonWidth}
         >
-          Refuse to vote
+          Abstain from voting
         </VotingButton>
       </div>
       <RefuseToVoteHint compactMode={compactMode} width={buttonWidth} />
```

Comment:
maybe if is used only here we can change this component name to `AbstainToVoteHint` ?